augly/video/helpers/ffmpeg.py

```diff
@@ -35,13 +37,25 @@ def combine_frames_and_audio_to_file(
 
 def extract_audio_to_file(video_path: str, output_audio_path: str) -> None:
     audio_info = get_audio_info(video_path)
-
-    (
-        ffmpeg.input(video_path, loglevel="quiet")
-        .output(output_audio_path, acodec=audio_info["codec_name"], ac=1)
-        .overwrite_output()
-        .run(cmd=FFMPEG_PATH)
-    )
+    sample_rate = str(audio_info["sample_rate"])
```

Comment:
You can pull & rebase onto main now since these changes are merged from https://github.com/facebookresearch/AugLy/pull/135!

Reply:
ignoring this file for review, as I already reviewed Zoe's diff :) please rebase :D